src/main/java/org/assertj/core/presentation/StandardRepresentation.java

```diff
@@ -630,19 +631,19 @@ private static String representGroup(List<String> representedElements, String st
 
   private String toStringOf(ChangeDelta<?> changeDelta) {
     return String.format("Changed content at line %s:%nexpecting:%n  %s%nbut was:%n  %s%n",
-                         changeDelta.lineNumber(),
-                         formatLines(changeDelta.getOriginal().getLines()),
-                         formatLines(changeDelta.getRevised().getLines()));
+      changeDelta.lineNumber(),
+      formatLines(changeDelta.getOriginal().getLines()),
+      formatLines(changeDelta.getRevised().getLines()));
```

Comment:
use assertj formatting preferences to avoid these kind of changes, the formatting preferences are in https://github.com/assertj/assertj-core/tree/main/src/ide-support